### ISSUE TITLE
Validation Loss did not decrease in the HMDB51 notebook?

### ISSUE THREAD

**Issue by neonampv** [Atze00/MoViNet-pytorch#25]
I trained your HMDB51 notebook in 10 epochs but the validation loss did not decrease? Why did it happened?

**Comment by Atze00** [comment]
Can you provide more information? Maybe a log of the training.
It's probably due to the fact that it converges from the first epoch

**Comment by nguyenquibk1996** [comment]
I had the same problem.
When I trained Movinet with my own dataset. The training loss decreased but the validation loss increased from the first epoch.
( see the below image )
If I trained Movinet not using pretrained Kinetics with HMDB51 in the notebook sample and my own dataset (i did not save a log of the training), both losses had not decreased.
Can you explain it? Thank you
![movinet_val_loss](https://user-images.githubusercontent.com/47473873/144568175-80606765-9080-4696-9654-c9af5d31e847.png)

**Comment by Atze00** [comment]
The notebook functions correctly, also I use the networks daily. It's unlikely these are problem related to part of the code of this repository. If you provide a colab short script that reproduces the problem I will look at it.

**Comment by nguyenquibk1996** [comment]
I will run your notebook with HMDB51 for 10 epochs and show to you a log of the training. Because I used the same function for my own dataset and got the same problem. I don't think  it can converges from the first epoch with many datasets.  When you train Movinet with your dataset, the validation loss decreases or not?
Thank you

**Comment by haowei2020** [comment]
> I trained your HMDB51 notebook in 10 epochs but the validation loss did not decrease? Why did it happened?

I want to train the HMDB51 notebook, but there is no define of transforms in line 8. How can you run this HMDB51 notebook successfully? 
![image](https://user-images.githubusercontent.com/82873816/162872805-be6e2051-4d15-429c-9156-0af8c5c29215.png)

**Comment by poincarelee** [comment]
I trained HMDB51 dataset for 20 epochs with modelA0_stream_statedict_v3, the result is as follows:
![image](https://user-images.githubusercontent.com/38481402/188830746-e881ea8a-1adb-49c0-ad21-22f45289232c.png)

**Comment by poincarelee** [comment]
@nguyenquibk1996 
Hi， did you solve the problem? I met the same problem in my own dataset.  Then I tried to train hmdb51 **without pretrained**, the evaluation accuracy is as follows:
![image](https://user-images.githubusercontent.com/38481402/197953216-a667c032-9bad-498d-8f5d-bc4622b61860.png)

Did I miss any key points during finetuning or could you give any clues about this?

**Comment by haowei2020** [comment]
> @nguyenquibk1996 Hi， did you solve the problem? I met the same problem in my own dataset. Then I tried to train hmdb51 **without pretrained**, the evaluation accuracy is as follows: ![image](https://user-images.githubusercontent.com/38481402/197953216-a667c032-9bad-498d-8f5d-bc4622b61860.png)
> 
> Did I miss any key points during finetuning or could you give any clues about this?

I think the dataset is primary cause, and the data processing method(one clip or multiple clips sampled from one video) is second cause. I have train the X3D and SlowFast on the HMDB51 by mmaction2(default config samples one clip from one video), top1 acc is also about 30%, and validation loss can decrease.

**Comment by poincarelee** [comment]
Have you tried any other datasets? BTW, I see in mmaction2 there's no training codes of X3D, could you tell me how to get the codes?

**Comment by haowei2020** [comment]
I have tried UCF101, the top1 can over 45%. I write the  codes of X3D by myself(of course refer to existing codes )

**Comment by haowei2020** [comment]
> I have tried UCF101, the top1 can over 45%. I write the codes of X3D by myself(of course refer to existing codes )

I have tried UCF101, the top1 can over 45%. I write the codes of X3D by myself(of course refer to existing codes )

**Comment by haowei2020** [comment]
> Have you tried any other datasets? BTW, I see in mmaction2 there's no training codes of X3D, could you tell me how to get the codes?

I have tried UCF101, the top1 can over 45%. I write the codes of X3D by myself(of course refer to existing codes )
